scripts/create-package.js

```diff
@@ -52,30 +52,102 @@ const experimentsPackageJson = JSON.parse(fs.readFileSync(experimentsPackagePath
 
 // Steps (mustache template names and output file paths)
 const steps = [
-  { template: 'NpmIgnore', output: '.npmignore' },
```

Comment:
My prettier extension may have caused this. Will clean it up in the AM 😴.

Reply:
I wouldn't expect Prettier to make this particular change--maybe it was the default VS Code JS formatter or some other formatting extension?

Reply:
Resolved in latest 3 commits